lib/daru/maths/statistics/vector.rb

```diff
@@ -87,12 +87,8 @@ def max_index
         end
 
         def frequencies
-          @data.inject({}) do |hash, element|
-            unless element.nil?
-              hash[element] ||= 0
-              hash[element] += 1
-            end
-            hash
+          @data.each_with_object(Hash.new(0)) do |element, hash|
+            hash[element] += 1 unless element.nil?
```

Comment:
It went to default Hash value (`each_with_object(Hash.new(0))`). 
As far as I can see, effect should be exactly the same, and no specs were broken.